src/Vehicle/Vehicle.cc

```diff
@@ -1728,8 +1728,38 @@ void Vehicle::_handlePing(LinkInterface* link, mavlink_message_t& message)
     sendMessageOnLink(link, msg);
 }
 
+void Vehicle::_handleACEHeartbeat(mavlink_message_t& message)
+{
+   int newACECustomModeIdx = (int)mavlink_msg_heartbeat_get_custom_mode(&message);
+   QString newACEMode;
+   switch(newACECustomModeIdx) {
+       case 0: newACEMode = "ACE:Idle";    break;
```

Comment:
we could use the same enum that defines the mode as the camera stuff:
https://github.com/planckaero/qgroundcontrol/blob/2cd809df0c9181daef0645f6599e8f9547698337/src/Camera/QGCCameraControl.h#L141-L149